cypress/utils/base_constants.js

```diff
@@ -5,3 +5,7 @@
 
 export const BASE_PATH = Cypress.config('baseUrl');
 export const BACKEND_BASE_PATH = Cypress.env('openSearchUrl');
+export const ADMIN_AUTH = {
```

Comment:
You can use the injected constants already defined [here](https://github.com/opensearch-project/opensearch-dashboards-functional-test/blob/main/cypress/utils/commands.js#L8)

Reply:
In fact, maybe export it in `base_constants` and then just reference it in `commands`?

Reply:
yeah, lets clean up the duplicate ADMIN_AUTH definition, thanks

Reply:
Sure, missed that it already exists.  Will remove